source/community/reactnative/src/visualizer/payloads/payloads/Action.NoBody.json

```diff
@@ -72,7 +72,7 @@
           "isMultiline": true,
           "inlineAction": {
               "type": "Action.Submit",
-              "iconUrl": "http://adaptivecards.io/content/cats/1.png",
+              "url": "http://adaptivecards.io/content/cats/1.png",
```

Comment:
It is iconUrl for action elements. Please refer to the schema - https://adaptivecards.io/explorer/Action.OpenUrl.html